### ISSUE TITLE
Using WorldEdit commands kicks client with "Internal Exception: io.netty.handler.codec.DecoderException"

### ISSUE THREAD

**Issue by RalphORama** [EngineHub/WorldEdit#1562]
#### WorldEdit version:

```
[23:08:32 INFO]: WorldEdit version 7.2.0-rc-01
[23:08:32 INFO]: https://github.com/EngineHub/WorldEdit/
[23:08:32 INFO]: ------------------ Platforms ------------------
* Bukkit-Official(7.2.0-rc-01+81f978e)

[23:08:32 INFO]: ---------------- Capabilities ----------------
GAME_HOOKS: Bukkit-Official
CONFIGURATION: Bukkit-Official
USER_COMMANDS: Bukkit-Official
PERMISSIONS: Bukkit-Official
WORLDEDIT_CUI: Bukkit-Official
WORLD_EDITING: Bukkit-Official
```

#### Platform version:

```
[23:09:39 INFO]: This server is running Paper version git-Paper-230 (MC: 1.16.3) (Implementing API version 1.16.3-R0.1-SNAPSHOT)
```

## Describe the bug

When issuing any WorldEdit command or setting positions with the wand item, the client who issued the command is kicked with the error `Internal Exception: 

```
io.netty.handler.codec.DecoderException: com.google.gson.JonsParseException: Don't know how to turn {"component": { ... }} into a Component
```

## To Reproduce

1. Install WorldEdit on PaperMC 1.16.3
2. Join server
3. Issue any WorldEdit command
4. The client is kicked.

## Expected behavior

Client shouldn't be kicked with an error.

#### `//wand`

![image](https://user-images.githubusercontent.com/6566104/96055257-8ddad200-0e51-11eb-9c69-b3d4db86ec4d.png)

#### `//brush sphere`

![image](https://user-images.githubusercontent.com/6566104/96055279-9c28ee00-0e51-11eb-9a14-e4a23a6f6fc2.png)

#### `//pos1`

![image](https://user-images.githubusercontent.com/6566104/96055291-a6e38300-0e51-11eb-90af-7d07dd84fd32.png)

## Additional Context

WorldEdit was working fine on my server when using WorldEdit 7.2.0 Beta 5, after upgrading to 7.2.0 RC1 the issue appeared. No error appears in the server console.

#### Server plugins:

```
[23:15:54 INFO]: Plugins (55): AngelChest, BlockLocker, BuycraftX, Chunkmaster, CommandPanels, ConsoleSpamFix, CoreProtect, DeathMessagesPrime, DiscordSRV, DragonDeathDrops*, DriveBackupV2*, EmojiChat, EmotePlugin3*, Essentials, EssentialsAntiBuild, EssentialsChat, EssentialsGeoIP, EssentialsProtect, EssentialsSpawn, ExtraContexts, FastLeafDecay, GameModeInventories, GSit, HiddenLorax, HoneyPot, IllegalStack, LimitPillagers, LuckPerms, MarriageMaster, Matrix, mcMMO, MessageAnnouncer, MoreMeat, OpenInv, Plan, PlayerHeads, ProtocolLib, PvPManager, SAML, SelectionVisualizer, ServerListPlus*, Shopkeepers, ShowItem, spark, StackMob, TimeControl, Vault, ViaBackwards, ViaVersion, VillagerModifications, VillagerOptimiser, WorldEdit, WorldGuard, WorldGuardExtraFlags, WorldResourcepacks
```

#### Client version:

- Minecraft 1.16.3
- Forge 34.1.0
- Client mods:

```
AppleSkin-mc1.16.2-forge-1.0.14.jar, ClientTweaks_1.16.3-5.2.0.jar, cloth-config-forge-4.1.0.jar, jei-1.16.3-7.5.0.43.jar, JustEnoughResources-1.16.3-0.12.0.100.jar, light-overlay-5.5.3.jar, MouseTweaks-2.13-mc1.16.2.jar, OptiFine_1.16.3_HD_U_G3.jar, ReAuth-1.16-Forge-3.9.2.jar, Xaeros_Minimap_20.25.0_Forge_1.16.3.jar, XaerosWorldMap_1.11.0_Forge_1.16.3.jar
```

**Comment by octylFractal** [comment]
Can you reproduce after restart? If not, we know this bug, but it's basically impossible to actually reproduce and figure out what's wrong, we've basically marked it as "unfixable".

**Comment by RalphORama** [comment]
Yup, a restart fixed this. Very strange.

**Comment by octylFractal** [comment]
Yup, it's a super strange bug that's gotten many reports but is seemingly impossible. Closing as this will likely be fixed when we move to adventure in #1480 anyways.
